spec/lib/dttp/params/placement_assignment_spec.rb

```diff
@@ -250,6 +251,46 @@ module Params
             })
           end
         end
+
+        context "funding", feature_show_funding: true do
```

Comment:
does having `feature_show_funding` off have any effect here?

Reply:
i wanted to specifically test that funding info would not be sent to DTTP even though we were collecting it in the front end i.e. test the DTTP related feature flag in this method:

```
      def send_funding_to_dttp?
        FeatureService.enabled?("show_funding") && FeatureService.enabled?("send_funding_to_dttp")
      end
```